app/GraphQL/Ecosystem/Mutations/Users/Update.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\GraphQL\Ecosystem\Mutations\Users;
+
+use GraphQL\Type\Definition\ResolveInfo;
+use Kanvas\Auth\Services\UserManagement as UserManagementService;
+use Kanvas\Users\Models\Users;
+use Nuwave\Lighthouse\Support\Contracts\GraphQLContext;
+
+class Update
```

Comment:
@arfenis move all user mutation to https://github.com/bakaphp/kanvas-ecosystem-api/blob/update-user-data/app/GraphQL/Ecosystem/Mutations/Users/UserManagement.php